pt_miniscreen/pages/templates/action/page.py

```diff
@@ -107,7 +107,7 @@ def action_state(self, state: ActionState) -> None:
 
         self.status_icon_hotspot.action_state = state
 
-    def on_select_press(self) -> None:
+    def start_action(self) -> None:
```

Comment:
good rename but doesn't this have to be updated at https://github.com/pi-top/pi-top-4-Miniscreen/blob/8d9eee46372dc97d0fcc49cb7599952033f2b94b/pt_miniscreen/tiles/templates/menu.py#L45

Reply:
yup! good catch thank you :)

Reply:
All fixed 👍